api.go

```diff
@@ -131,8 +98,8 @@ func Server() {
 			}
 
 			// Prepare request object to send files from form to bucket
-			bucketUrl := "https://kasxttiggmakvprevgrp.supabase.co/storage/v1/object/assignments/" + assignmentId + "/"
-
+			bucketUrl := supabase.BaseURL + "/" + "storage/v1/object/assignments/" + assignmentId + "/"
```

Comment:
nit- can probably combine the `"/"` and `"storage[...]"`

Reply:
cant believe ive done this...